content/en/docs/labs/boot/getting-started/cloud/gcloud.md

```diff
@@ -44,6 +44,13 @@ cd cloud-resources/gcloud
 ./create_cluster.sh
 ```
 
+<nav>
+  <ul class="pagination">
```

Comment:
Should we really start messing with the built-in navigation? What is your concern here? That the page is so long and the top breadcrumb is gone? Maybe better in this case to split the page?